packages/operator/src/main/java/io/spaship/operator/service/k8s/Operator.java

```diff
@@ -398,5 +398,18 @@ private void processK8sList(KubernetesList result, UUID tracing, String nameSpac
 
   }
 
+  public ConfigMap updateConfigMap(Environment environment, Object syncConfig){
+
+    var configMapName = "sidecar-config-"
```

Comment:
sidecar-config-

can't it be only "sidecar-config" !

Reply:
okay